arcgis-ios-sdk-samples/Edit data/Create and save KML file/CreateAndSaveKMLViewController.swift

```diff
@@ -0,0 +1,270 @@
+// Copyright © 2020 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class CreateAndSaveKMLViewController: UIViewController {
+    // Set the map.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap(basemap: .darkGrayCanvasVector())
+            mapView.sketchEditor = AGSSketchEditor()
+        }
+    }
+    
+    @IBOutlet var addButton: UIBarButtonItem!
+    @IBOutlet var sketchDoneButton: UIBarButtonItem!
+    @IBOutlet var toolbar: UIToolbar!
+    @IBOutlet var actionButtonItem: UIBarButtonItem!
+    @IBOutlet var resetButtonItem: UIBarButtonItem!
+    
+    // Prompt feature selection action sheet.
+    @IBAction func addFeature() {
+        let alertController = UIAlertController(title: "Select Feature", message: nil, preferredStyle: .actionSheet)
+        let pointAction = UIAlertAction(title: "Point", style: .default) { (_) in
+            self.addPoint()
+        }
+        alertController.addAction(pointAction)
+        let polylineAction = UIAlertAction(title: "Polyline", style: .default) { (_) in
+            self.addPolyline()
+        }
+        alertController.addAction(polylineAction)
+        let polygonAction = UIAlertAction(title: "Polygon", style: .default) { (_) in
+            self.addPolygon()
+        }
+        alertController.addAction(polygonAction)
+        
+        // Add "cancel" item.
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
```

Comment:
Cancel button is displayed only for iPhone form factor but not for iPad form factor. Is this an expected behavior?

Reply:
See the [HIG](https://developer.apple.com/design/human-interface-guidelines/ios/views/popovers/):

> A Close button, such as Cancel or Done, is worth including if it provides clarity, such as exiting with or without saving changes. In general, a popover should close automatically when its presence is no longer necessary. In most cases, a popover should close when someone taps outside of its bounds or selects an item in the popover.